composer/loggers/mlflow_logger.py

```diff
@@ -528,21 +608,49 @@ def log_images(
 
     def post_close(self):
         if self._enabled:
+            if hasattr(self, 'monitor_process'):
+                # Check if there is an uncaught exception, which means `post_close()` is triggered
+                # due to program crash.
+                if isinstance(self._global_exception_occurred, torch.Tensor):
+                    finish_with_exception = (self._global_exception_occurred == 1).item()
+                else:
+                    finish_with_exception = (self._global_exception_occurred == 1)
```

Comment:
can we simplify to use a bool flag and get rid of the tensor/move to device stuff

Reply:
Yes I thought about that, right now I am using a `torch.Tensor` because to resolve the single node multi-gpu setup, we need to communicate the `self._global_exception_occurred` across GPUs. If we don't plan to look into that setup in a short term, happy to change it to a bool variable!